### PR TITLE
comments, round one

### PR DESCRIPTION
**Pull request by ghost**
Hi @nap84 ,

there are some comments so far. 
I will continue later on.

### CODE REVIEW

File: docs/40_topics/01_fair.mdx [hunk 1]
```diff
@@ -7,37 +7,90 @@ author: "Nicole A. Parks"
...
+the FAIR Data Principles place a strong focus on machine-readable (meta)data.
+While humans must be able to find and understand the published datasets as well,
+computers will be tasked with sifting through large amounts of data and
+determining what datasets are relevant to the intended purpose and how they can be reused.
+To this end, not only is it important that humans are able to understand
+and find the data but, more importantly,
+machines need to be able to search through available datasets and
+determine which ones are of use for the intended purpose.
+
+<!--Der nächste Absatz ist an sich nicht speziell auf die
```
Comment (nap84) [inline docs/40_topics/01_fair.mdx:26]: Hast Recht!

File: docs/40_topics/01_fair.mdx [hunk 2]
```diff
@@ -7,37 +7,90 @@ author: "Nicole A. Parks"
...
+This type of identifier ensures the object it refers to,
+such as a dataset or a journal publication,
+are not only findable but also citable.
+
+<!-- Ich würde hier nochmals herausstellen, was ein PID so besonders
+und für die Forschung essentiell macht: Die Garantie der
+PID-Servicestellen, dass die hinter einer PID/DOI hinterlegten Daten
+stets verfügbar und referenzierbar sind und dass sich die Daten
+nicht ändern (ansonsten gibt es eine Version und eine neue PID/DOI).-->
+
```
Comment (nap84) [inline docs/40_topics/01_fair.mdx:70]: Stimmt das aber immer? Beispielsweise, in dem Fall von [accepted manuscripts bei der RSC](https://www.rsc.org/journals-books-databases/author-and-reviewer-hub/process-and-policies/#accepted-manuscripts) wird ein DOI für die accepted Version erteilt, derselbe aber auch für die finale Version benutzt.

**Comment by ghost** [comment]
consider especially https://github.com/nap84/knowledge_base/pull/1/commits/e5b89807db4307df5dce7bc33a1000dc5e4e336f

There are no links internally? However you mention and refer to certain headlines, you could also include internal links. 

Last section  (R) to be followed.

**Comment by ghost** [comment]
I completed the reading. Well done. 
Just some minor remarks.

**Pull request closed without merging**